### DISCUSSION TITLE
What is the unity used in fetal and mature Kidney scRNAseq,

### DISCUSSION THREAD

**Discussion by margueritehureaux** [chanzuckerberg/cellxgene#2474] [Q&A]
Please, excuse-me for this question that might have been already solved. 
I read all the tutorial of the Atlas tool, but I cannot find the answer of:
- what is the unity of the data proposed on the graph under the genes names? on the right (0 is in green and more expression is 3 or 4 in blue), FPFM? or other unity?
- How can I filter to see only the cells that express more than 3 (unity?), and to see at what type of cell it correspond?
Thanks you again,
Dr M Hureaux, geneticist in Paris APHP

**Comment by MaximilianLombardo** [comment]
Hi Marguerite,

Thanks for your question!

> what is the unity of the data proposed on the graph under the genes names? on the right (0 is in green and more expression is 3 or 4 in blue), FPFM? or other unity?

The units that are displayed are dataset dependent so if you have a specific dataset that you are asking about this for, then it would be best to refer to the author's paper and see what kind of normalization they had performed - with the numbers that you had given - I'm guessing that these have been log normalized

> How can I filter to see only the cells that express more than 3 (unity?), and to see at what type of cell it correspond?

You can do this by brushing the histogram (this will select cells based on their expression level) and subsetting that selection. Then, with the gene of interest colored, you can expand the cell type metadata field to view the distribution of this gene's expression (i.e. look to see if there are any cell types that tend to express the gene at a higher level)

Let me know if that helps or if you need any further clarification,

Max
